examples/stocks/tests/locale_test.dart

```diff
@@ -0,0 +1,23 @@
+
+import 'package:test/test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:stocks/main.dart' as stocks;
+import 'package:stocks/stock_data.dart' as stock_data;
+
+void main() {
+  stock_data.StockDataFetcher.actuallyFetchData = false;
+
+  test("Test changing locale", () {
+    testWidgets((WidgetTester tester) {
+      stocks.main();
+      tester.pump(); // Start startup animation
+      tester.pump(const Duration(seconds: 1)); // Complete startup animation
```

Comment:
There shouldn't be a startup animation anymore.  Does the test fail if you just pump once?